src/Compilers/Core/Portable/Symbols/IEventSymbol.cs

```diff
@@ -19,6 +19,11 @@ public interface IEventSymbol : ISymbol
         /// </summary>
         ITypeSymbol Type { get; }
 
+        /// <summary>
+        /// The declareted nullability of the this event.
+        /// </summary>
+        Nullability DeclaredNullability { get; }
```

Comment:
I don't like the naming here.  If type field is called `ITypeSymbol Type` it feels like this should be `Nullability TypeNullability`.  That way you can easily track what this is referring to.